packages/features/bookings/Booker/utils/event.ts

```diff
@@ -112,7 +111,7 @@ export const useScheduleForEvent = ({
     rescheduleUid,
     month: monthFromStore ?? month,
     duration: durationFromStore ?? duration,
-    isTeamEvent: pathname?.indexOf("/team/") !== -1 || isTeam,
```

Comment:
This was causing duplicate request of getSchedule. isTeam was intially false but later it was calculated to be true for team event types